api4/cloud.go

```diff
@@ -393,3 +395,30 @@ func handleCWSWebhook(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	ReturnStatusOK(w)
 }
+
+func sendOverLimitAlert(c *Context, w http.ResponseWriter, r *http.Request) {
+	if c.App.Srv().License() == nil || !*c.App.Srv().License().Features.Cloud {
+		c.Err = model.NewAppError("Api4.sendOverLimitAlert", "api.cloud.license_error", nil, "", http.StatusNotImplemented)
+		return
+	}
+
+	user, err := c.App.GetUser(c.App.Session().UserId)
+	if err != nil {
+		c.Err = err
+		return
+	}
+
+	sub, subErr := c.App.Cloud().GetSubscription()
```

Comment:
Why not just err here for consistency?

```suggestion
	sub, err := c.App.Cloud().GetSubscription()
```